modules/installation-mirror-repository.adoc

```diff
@@ -104,6 +104,8 @@ $ REMOVABLE_MEDIA_PATH=<path> <1>
 <1> Specify the full path, including the initial forward slash (/) character.
 
 . Mirror the version images to the internal container registry:
++
```

Comment:
If the note is intended to be an admonition for the whole "Mirror the version images..." step, then this is a way to do so that keeps the note at the same level as this step.

Otherwise, if the note is only intended to comment on the "Record the entire `imageContentSources`..." step, then it could just be a simple `+` without the wrapping `--` open block.